src/tools/dartpad/troubleshoot.md

```diff
@@ -40,10 +40,41 @@ Although DartPad doesn't use cookies, it does rely on local storage,
 which browsers usually disable when cookies are disabled.
 
 
+## Code doesn't work outside DartPad
+
+If code from DartPad does not run as expected in other environments,
+check the following:
+
+* If you're creating a web app,
```

Comment:
Let's make the web app the last one, since it's probably the most complex and least common case.